scipy/stats/tests/test_stats.py

```diff
@@ -3632,24 +3627,25 @@ class SkewKurtosisTest:
     testcase = [1., 2., 3., 4.]
     testmathworks = [1.165, 0.6268, 0.0751, 0.3516, -0.6965]
 
-
-@make_skip_xp_backends(stats.skew)
-class TestSkew(SkewKurtosisTest):
-    @make_skip_xp_backends(stats.kurtosis)
-    @pytest.mark.parametrize('stat_fun', [stats.skew, stats.kurtosis])
-    def test_empty_1d(self, stat_fun, xp):
+    def test_empty_1d(self, xp):
         x = xp.asarray([])
         if is_numpy(xp):
             with pytest.warns(SmallSampleWarning, match=too_small_1d_not_omit):
-                res = stat_fun(x)
+                res = self.stat_fun(x)
         else:
             with np.testing.suppress_warnings() as sup:
                 # array_api_strict produces these
                 sup.filter(RuntimeWarning, "Mean of empty slice")
                 sup.filter(RuntimeWarning, "invalid value encountered")
-                res = stat_fun(x)
+                res = self.stat_fun(x)
         xp_assert_equal(res, xp.asarray(xp.nan))
 
+
+@make_skip_xp_backends(stats.skew)
+class TestSkew(SkewKurtosisTest):
+    def stat_fun(self, x):
```

Comment:
With `stat_fun = stats.skew`, `stats.skew` was being passed `self` as the first argument.